run-tests

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+EXPECT_5LVL=0
+SUPPORT_NO5LVL=0
+SUPPORT_LA57=0
+SUPPORT_5LVL=0
+
```

Comment:
Suggestion: move the `SUPPORT_*` variables down under the "Detect configuration" comments.  While `EXPECT_5LVL` is a "global variable", the `SUPPORT_*` variables are probably going to be only local to the configuration detection code.